azuredeploy.json

```diff
@@ -569,255 +576,258 @@
-                "parameters": {
-                    "moodleCommon": {
-                        "value": "[variables('moodleCommon')]"
-                    },
-                    "dbFQDN": {
-                        "value": "[reference('dbTemplate').outputs.dbFQDN.value]"
-                    },
-                    "subnetIdWeb": {
-                        "value": "[reference('networkTemplate').outputs.subnetIdWeb.value]"
-                    },
-                    "ctlrPubIpId": {
-                        "value": "[reference('networkTemplate').outputs.ctlrPubIpId.value]"
-                    },
-                    "redisKey": {
-                        "value": "[if(parameters('redisDeploySwitch'), reference('redisTemplate').outputs.redisKey.value, 'None')]"
-                    },
-                    "azureSearchKey": {
-                        "value": "[if(equals(parameters('searchType'), 'azure'), reference('searchTemplate').outputs.azureSearchKey.value, 'None')]"
-                    }
-                },
-                "templateLink": {
-                    "uri": "[concat(variables('moodleCommon').baseTemplateUrl,'controller.json',parameters('_artifactsLocationSasToken'))]"
-                }
-            }
-        },
-        {
-            "type": "Microsoft.Resources/deployments",
-            "apiVersion": "2017-05-10",
-            "dependsOn": [
-                "Microsoft.Resources/deployments/controllerTemplate",
-                "Microsoft.Resources/deployments/networkTemplate",
-                "Microsoft.Resources/deployments/redisTemplate",
-                "Microsoft.Resources/deployments/dbTemplate"
-            ],
-            "name": "scaleSetTemplate",
-            "properties": {
-                "mode": "Incremental",
-                "parameters": {
-                    "moodleCommon": {
-                        "value": "[variables('moodleCommon')]"
-                    },
-                    "subnetIdWeb": {
-                        "value": "[reference('networkTemplate').outputs.subnetIdWeb.value]"
-                    }
-                },
-                "templateLink": {
-                    "uri": "[concat(variables('moodleCommon').baseTemplateUrl,'webvmss.json',parameters('_artifactsLocationSasToken'))]"
-                }
-            }
-        },
-        {
-            "type": "Microsoft.Resources/deployments",
-            "apiVersion": "2017-05-10",
-            "name": "storageAccountTemplate",
-            "properties": {
-                "mode": "Incremental",
-                "parameters": {
-                    "moodleCommon": {
-                        "value": "[variables('moodleCommon')]"
-                    }
-                },
-                "templateLink": {
-                    "uri": "[concat(variables('moodleCommon').baseTemplateUrl,'storageAccount.json',parameters('_artifactsLocationSasToken'))]"
-                }
-            }
+  "resources": [
```

Comment:
The rest of these changes seem to be white space only. That shouldn't happen. Did the original file have tabs? Does this new one have tabs? Something else? 

Whatever, we shouldn't have all these white space changes, it's very hard to review these changes.